input/pagecontent/auereqdi.md

```diff
@@ -33,216 +33,216 @@ Column attribute descriptions are as follows:
   <tr>
     <td rowspan="9">Service request</td>
     <td colspan="2">Service name</td>
-    <td>TBD</td>
-    <td>TBD</td>
+    <td><a href="StructureDefinition-au-erequesting-servicerequest.html">AU eRequesting ServiceRequest</a></td>
+    <td>ServiceRequest.code</td>
     <td></td>
   </tr>
   <tr>
     <td colspan="2">Clinical indication</td>
-    <td>TBD</td>
-    <td>TBD</td>
+    <td><a href="StructureDefinition-au-erequesting-servicerequest.html">AU eRequesting ServiceRequest</a></td>
+    <td>ServiceRequest.reasonCode</td>
     <td></td>
   </tr>
   <tr>
     <td colspan="2">Clinical context</td>
-    <td>TBD</td>
-    <td>TBD</td>
+    <td><a href="StructureDefinition-au-erequesting-servicerequest.html">AU eRequesting ServiceRequest</a></td>
+    <td>ServiceRequest.note</td>
     <td></td>
   </tr>
   <tr>
     <td colspan="2">Urgency</td>
-    <td>TBD</td>
-    <td>TBD</td>
-    <td></td>
+    <td><a href="StructureDefinition-au-erequesting-servicerequest.html">AU eRequesting ServiceRequest</a></td>
+    <td>ServiceRequest.priority</td>
+    <td>This element is allowed but does not have a Must Support flag.</td>
   </tr>
    <tr>
     <td colspan="2">Service due</td>
-    <td>TBD</td>
-    <td>TBD</td>
-    <td></td>
+    <td><a href="StructureDefinition-au-erequesting-servicerequest.html">AU eRequesting ServiceRequest</a></td>
+    <td>ServiceRequest.occurrence[x]</td>
+    <td>This element is allowed but does not have a Must Support flag.<br><br>This element is choice between occurrenceTiming | occurrenceDateTime | occurrencePeriod.</td>
   </tr>
    <tr>
     <td colspan="2">Comment</td>
-    <td>TBD</td>
-    <td>TBD</td>
+    <td><a href="StructureDefinition-au-erequesting-servicerequest.html">AU eRequesting ServiceRequest</a></td>
+    <td>ServiceRequest.note</td>
     <td></td>
   </tr>
    <tr>
     <td colspan="2">Distribution list</td>
     <td>TBD</td>
     <td>TBD</td>
-    <td></td>
+    <td>Work is underway in AU eRequesting to map this element. Feedback is requested on the appropriateness of using either CommunicationRequest or an extension to support distribution list.</td>
```

Comment:
After every "Feedback is requested ..." Add Please comment by raising [HL7 Jira Issues](https://jira.hl7.org/projects/FHIR/issues).

Reply:
Done